core/server/master/src/test/java/alluxio/master/file/meta/MountTableTest.java

```diff
@@ -76,18 +76,12 @@ public void path() throws Exception {
           e.getMessage());
     }
 
-    try {
-      addMount("/mnt/bar/baz", "/baz", 5);
-    } catch (InvalidPathException e) {
-      // Exception expected
-      Assert.assertEquals(
-          ExceptionMessage.MOUNT_POINT_PREFIX_OF_ANOTHER.getMessage("/mnt/bar", "/mnt/bar/baz"),
-          e.getMessage());
-    }
+    // Testing nested mount
```

Comment:
can we make a nested mount test a separate test method?

Reply:
done.